test/new-e2e/agent-subcommands/flare/flare_test.go

```diff
@@ -27,10 +29,15 @@ func TestFlareSuite(t *testing.T) {
 }
 
 func requestAgentFlareAndFetchFromFakeIntake(v *commandFlareSuite, flareArgs ...client.AgentArgsOption) flare.Flare {
+	// Wait for the fakeintake to be ready to avoid 503 when sending the flare
+	v.EventuallyWithT(func(c *assert.CollectT) {
```

Comment:
@pducolin : What do you think about moving this code to `test-infra-definition` to make sure fakeintake is ready when the user have access to `v.Env().Fakeintake`?